utils/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I'm curious why these new files are in `utils` instead of `api/_lib/script`?

Reply:
That's a good point, thank you.